jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -231,19 +251,13 @@ private Directory attachDownloadsDirectory() {
 		return directory;
 	}
 
-	private void attachReports(RootRouter router)
-			throws ConfigurationException, ValidationException {
+	private void attachReports(RootRouter router) throws ConfigurationException, ValidationException {
 		router.attach("/reports", new ReportRouter(getContext(), router.getApplication()));
 	}
```

Comment:
I checked the code of `ReportRouter` and this code is not similar to all others resources because it adds a `Authorizer` and a `Filter`.
So I don't know if&how we can safely extract (all | part of) the content of the `ReportRouter` to `attachReports()`.